src/Message/CaptureRequest.php

```diff
@@ -116,6 +117,38 @@ private function isPaymentCaptured(stdClass $payment): bool
         return isset($payment->authorization->capture);
     }
 
+    /**
+     * Modifies a erroneous capture response to successful when the payments are already captured.
+     *
+     * This is required to facilitate the same workflow for both directly captured payment methods (like iDeal)
+     * and delayed captured payment methods (like ELV).
+     *
+     * @param stdClass $captureResponse
+     * @param stdClass $statusResponse
+     */
+    private function modifyCaptureResponseToSuccessfulWhenAlreadyCaptured(
+        stdClass $captureResponse,
+        stdClass $statusResponse
+    ): void {
+        if (isset($captureResponse->captureSuccess)) {
+            return;
+        }
+
+        if (isset($statusResponse->statusSuccess) === false) {
+            return;
+        }
+
+        $statusResponseApproximateTotals = $statusResponse->statusSuccess->report->approximateTotals;
+        if ($statusResponseApproximateTotals->totalRegistered !== $statusResponseApproximateTotals->totalCaptured) {
```

Comment:
The amount of `totalCaptured` cannot be more than `totalRegistered`?

Reply:
No, unless something is wrong on the side of CM Payments. And then I think you'd want the transaction be unsuccessful.